wzdx/standard_to_enhanced/navjoy_translator.py

```diff
@@ -2,8 +2,9 @@
 import copy
```

Comment:
We're a little light on code coverage with this one, did we want to boost that up?

Reply:
I added a test, but all that is left over is the main method (which is annoying to test and is not run in the cloud), and a bunch of individual lines of code. I can increase it more if you want, but it is at 84% right now